debezium-connector-sqlserver/src/main/java/io/debezium/connector/sqlserver/SqlServerConnectorConfig.java

```diff
@@ -365,6 +366,7 @@ public static ConfigDef configDef() {
 
     private final String databaseName;
     private final String instanceName;
+    private final String skippedOperations;
```

Comment:
Was there some additional intent for storing this locally?  I see we set this in the constructor but from what I see, its never used.  If there isn't a need for it, we should probably remove it.